riscv/tests/riscv_data/vec_median/Cargo.toml

```diff
@@ -4,6 +4,6 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-runtime = { path = "../../../runtime" }
+powdr-riscv-rt = { path = "../../../../powdr_riscv_rt" }
```

Comment:
I would opt for actually spelling `runtime` out. `rt` can mean so many things.

Reply:
I think this is rather common. We use another crate called riscv_rt for qemu, so i thought powdr-riscv-rt fits well

Reply:
I mean, common also if you include the context you get from the "riscv" prefix to "rt"

Reply:
Even if it's common, I really don't think it's a good idea. We also don't abbreviate executor or other stuff.

Reply:
I mean isn't even runtime already an abbreviation for runtime environment?

Reply:
Ok. I still have a somewhat strong opinion about this. `powdr-riscv-runtime` is just really long and kinda clumsy. If someone else says they prefer it spelled out I'll give in.

Reply:
What does it matter if it's long and clumsy? Do you need to type it that often?

Reply:
just aesthetics